app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -103,7 +103,7 @@ ListingController = (
   $scope.imageURL = (listing) ->
     # TODO: remove "or" case when we know we have real images
     # just a fallback for now
-    listing.Building_URL || 'https://placehold.it/474x316'
+    listing.imageURL || 'https://placehold.it/474x316'
 
```

Comment:
@creativecoder should we finally just get rid of this function and refer directly to `listing.imageURL`? I think it's only referred to in one place anyway. And then I think our practice should be, if a listing layout is messed up because of no image (e.g. this sometimes happens on fake listings on QA) then just add an image in salesforce rather than relying on this placeholder to correct the layout.